web/app_options.js

```diff
@@ -103,6 +103,11 @@ const defaultOptions = {
     value: typeof PDFJSDev === "undefined" || !PDFJSDev.test("CHROME"),
     kind: OptionKind.VIEWER + OptionKind.PREFERENCE,
   },
+  enableStampEditor: {
+    /** @type {boolean} */
+    value: true,
+    kind: OptionKind.VIEWER + OptionKind.PREFERENCE,
+  },
```

Comment:
Is this option/preference intended to be *temporary* during development?
If so, perhaps add a comment to that affect?